docs/.vitepress/config.ts

```diff
@@ -0,0 +1,80 @@
+import { defineConfig } from 'vitepress'
+import { generateSidebar } from 'vitepress-sidebar'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  lang: 'fr',
+  title: "Documentation Recolt'Ô",
+  description: '',
+  srcDir: "content",
+  base: "/",
+  metaChunk: false,
+  head: [
+    ['link', {
+      rel: 'icon',
+      href: "/favicon.ico"
+    }],
+    ['meta', {
+      property: 'og:image',
+      content: 'https://raw.githubusercontent.com/makinacorpus/DbToolsBundle/main/docs/content/public/meta.png'
+    }]
+  ],
+  themeConfig: {
+    // https://vitepress.dev/reference/default-theme-config
+    logo: '/logo.svg',
+    nav: [
+      { text: 'Accueil', link: '/' },
+    ],
+    editLink: {
+      pattern: 'https://github.com/makinacorpus/recolto/blob/main/docs/content/:path',
+      text: 'Modifier cette page sur Github'
+    },
+    docFooter: {
+      prev: 'Page précédente',
+      next: 'Page suivante'
+    },
+    outlineTitle: 'Sur cette page',
+    lastUpdated: {
+      text: 'Dernière mise à jour',
+      formatOptions: {
+        dateStyle: 'short'
+      }
+    },
+    search: {
+      provider: 'local'
+    },
+    footer: {
+      message: 'Publié sous la licence MIT.',
+      copyright: 'Copyright © 2024-aujourd\'hui <a href="https://makina-corpus.com/territoires">Makina Corpus Territoire</a>'
+    },
+    socialLinks: [
+      { icon: 'github', link: 'https://github.com/makinacorpus/recolto' }
+    ],
+    sidebar: generateSidebar({
+      documentRootPath: 'content',
```

Comment:
I did make some tests right here: 
- with the suggested "content": got the following error:
```
failed to load config from /Users/…/dev/recolto-makina/docs/.vitepress/config.ts
failed to start server. error:
ENOENT: no such file or directory, scandir '/Users/gnome/dev/recolto-makina/content'
Error: ENOENT: no such file or directory, scandir '/Users/gnome/dev/recolto-makina/content'
```
- with "/" I had a working vitepress with a sidebar (including docs, README.md) but all individual pages were 404
- with "docs/content" I get a working sidebar only with the items in "content" + the pages, I assume that’s the expected result?

-> suggestion
```suggestion
      documentRootPath: 'docs/content',
```

Reply:
ok, my bad, my readme is not explicit enough :grimacing:  

You need to launch every commands in the `docs` folder.: `cd docs`. Then it should work properly